experiments/AMIP/modular/components/ocean/slab_seaice_init.jl

```diff
@@ -82,3 +96,15 @@ clean_sic(SIC, _info) = swap_space!(zeros(axes(_info.land_fraction)), SIC) ./ fl
 # setting that SIC < 0.5 is counted as ocean if binary remapping.
 get_ice_fraction(h_ice::FT, mono::Bool, threshold = 0.5) where {FT} =
     mono ? h_ice : Regridder.binary_mask(h_ice, threshold = FT(threshold))
+
+# required by Interfacer
+get_field(sim::PrescribedIceSimulation, ::Val{:air_temperature}) = sim.integrator.u.T_sfc
+get_field(sim::PrescribedIceSimulation, ::Val{:z0m}) = sim.integrator.p.params.z0m
+get_field(sim::PrescribedIceSimulation, ::Val{:z0b}) = sim.integrator.p.params.z0b
+get_field(sim::PrescribedIceSimulation, ::Val{:beta}) = convert(eltype(sim.integrator.u), 1.0)
+get_field(sim::PrescribedIceSimulation, ::Val{:albedo}) = sim.integrator.p.params.α
+get_field(sim::PrescribedIceSimulation, ::Val{:area_fraction}) = sim.integrator.p.area_fraction
+
+function update_field!(sim::PrescribedIceSimulation, ::Val{:area_fraction}, field::Fields.Field)
```

Comment:
Do we need an `update_field!` method for `air_temperature`?

Reply:
Not in this PR for atmos (this will be done in the next PR), and `air_temperature` here is actually the `prognostic T_sfc` in the surface models, so it's updated internally. I will change this name to `surface_temperature`, which is more accurate.